.github/workflows/nightly-backward-compatibility.yml

```diff
@@ -18,17 +19,42 @@ on:
         default: "10.1.1"
```

Comment:
```suggestion
        default: "10.1.2"
```

Reply:
Updated in this [commit](https://github.com/input-output-hk/mithril/pull/2083/commits/f6e0eb72fc860c9032d3198b18386ee6b6968003) ✅ .